export.py

```diff
@@ -0,0 +1,44 @@
+import StringIO
+import csv
+from flask import make_response, jsonify
+
+
+def to_csv(arrangements):
+    si = StringIO.StringIO()
+    cw = csv.writer(si)
+
+    cw.writerow(arrangements.keys())
```

Comment:
sure we can do this for now. But the actual way we want to write out the CSV is very different. But at least there is something in place for now.

Reply:
what's you suggestion on writing out the CSV?